huxley/www/js/stores/CurrentUserStore.js

```diff
@@ -36,13 +38,32 @@ class CurrentUserStore extends Store {
 
   updateSchool(schoolID, delta) {
     ServerAPI.updateSchool(schoolID, delta);
-    const user = this._currentUser.school;
+    const user = this._currentUser;
     this._currentUser = {
       ...user,
       school: {...user.school, ...delta},
     };
   }
 
+  updateUser(userID, delta) {
+    $.ajax({
+      type: 'PATCH',
+      url: '/api/users/' + userID,
+      data: JSON.stringify(delta),
+      contentType: 'application/json',
+    });
+
+    const user = {
+      ...this._currentUser,
+      first_name: delta.first_name,
```

Comment:
This assumes that the delta will always have first and last name, which is true for now, but may not always be. We don't need to change anything right now, but just keep it in mind.